src/modules/ROOT/pages/changelog/202209/02.json

```diff
@@ -0,0 +1,393 @@
+      ],
+      "how": [
+        "'.number' was appended to the password constraint's input model, translation strings' context was changed"
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-locale/commit/703516c9e23bf5373de01ba073b11426cbe66390",
+        "https://github.com/cortezaproject/corteza-webapp-admin/commit/a2480009e6f1c59c10569343bef98270ce0968d2"
+      ]
+    },
+    {
+      "what": [
+        "Documentation about sanitizers and fixed the example links in module field validation tab to now point to the proper anchors."
+      ],
+      "why": [
+        "Documentation was lacking information about field value sanitization."
+      ],
+      "how": [
+        "The documentation was updated with a new important section describing this behavior.",
+        "And documentation links in module field validators tab now point to validators/sanitizers respectively"
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-webapp-compose/commit/9af28f269ae4752c036522273ec9c1af6a14efd2"
+      ]
+    }
+  ],
+
+  "fixed": [
+    {
+      "what": [
+        "Incorrect user count from the system metrics report due to system-defined users"
+      ],
+      "how": [
+        "System users are now excluded from the metric report, we didn't exclude them from the total, but they were excluded from the per-day aggregation"
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-server/commit/fd6465d0f95d78210401f0d0be9c16e1290341af"
+      ]
+    },
+    {
+      "what": [
+        "The timestamp in the details of an action log being different as the one in search results"
+      ],
+      "how": [
+        "The custom format function IocFullDateTime was applied to the timestamp"
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-webapp-admin/commit/a4596fb283b42822fe3bb4c81c4cbc4f43d7bfe4"
+      ]
+    },
+    {
+      "what": [
+        "Strict record value duplicate detection, which was preventing record creation with same values for fields for strict duplicate detection validation"
+      ],
+      "how": [
+        "Validating duplication only for selected field"
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-server/commit/eac2f07"
+      ]
+    },
+    {
+      "what": [
+        "The issue caused by the .Load(...) workflow service function, not including sub-workflows which excluded them from the cache and made them unavailable in the workflow execution step."
+      ],
+      "how": [
+        "The patch now includes sub-workflows in the initial load thus resolving the issue."
+      ],
+      "refs": [
+        "https://github.com/cortezaproject/corteza-server/commit/d64b732c25273ca3b588cc94ebb716a1a0fabca7"
+      ]
+    },
+    {
+      "what": [
+        "The userID, once filtered by it, you could not remove the filter which made the server filter by 0 which will never exist."
+      ],
+      "how": [
```

Comment:
This how doesn't correspond to the above what? What talks about userID, how about timestamps